cmd/swapcli/main.go

```diff
@@ -290,6 +290,34 @@ func cliApp() *cli.App {
 					swapdPortFlag,
 				},
 			},
+			{
+				Name: "claim",
+				Usage: "manually call claim() in the contract for a given swap. " +
+					"WARNING: This should only be used if the normal swap process fails.",
+				Action: runClaim,
+				Flags: []cli.Flag{
+					&cli.StringFlag{
+						Name:     flagOfferID,
+						Usage:    "ID of swap for which to call claim()",
+						Required: true,
+					},
+					swapdPortFlag,
+				},
+			},
+			{
+				Name: "refund",
+				Usage: "manually call refund() in the contract for a given swap. " +
+					"WARNING: This should only be used if the normal swap process fails.",
```

Comment:
I just tested, and you can place newlines in these strings. `swapcli` will correctly indent after the newline.